### ISSUE TITLE
Travis CI failure due to poppler

### ISSUE THREAD

**Issue by quintusdias** [quintusdias/glymur#497]
Failures on 3.6 and 3.7.  Python 3.8 passes though.

```
$ python -m unittest discover
EEEEEEEEEEEEEE
======================================================================
ERROR: glymur (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: glymur
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/unittest/loader.py", line 462, in _find_test_path
    package = self._get_module_from_name(name)
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/travis/build/quintusdias/glymur/glymur/__init__.py", line 8, in <module>
    from .jp2k import Jp2k
  File "/home/travis/build/quintusdias/glymur/glymur/jp2k.py", line 26, in <module>
    from .jp2box import (
  File "/home/travis/build/quintusdias/glymur/glymur/jp2box.py", line 27, in <module>
    import gdal
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/site-packages/gdal.py", line 2, in <module>
    from osgeo.gdal import deprecation_warn
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/site-packages/osgeo/__init__.py", line 21, in <module>
    _gdal = swig_import_helper()
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/site-packages/osgeo/__init__.py", line 17, in swig_import_helper
    _mod = imp.load_module('_gdal', fp, pathname, description)
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/home/travis/miniconda/envs/test-environment/lib/python3.6/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: libpoppler.so.76: cannot open shared object file: No such file or directory
```

**Comment by quintusdias** [comment]
Have reproduced the Travis-CI issue on a bionic VM.  It would seem that the poppler package is providing  ``lib/libpoppler.so.78`` but gdal is looking for version 76.  Perhaps we can not use conda-forge anymore.  Not sure we need it.

**Comment by quintusdias** [comment]
Turns out that importlib_resources is not available for 3.6 thru the defaults channel, so the 3.6 jobs fail.  Adding conda-forge back, but appending it rather than prepending it results in successes across all platforms, but the 3.8 job looks like it only accidentally succeeded.  GDAL was not installed because it is not available via the defaults channel yet.  So, we need to actually move to a system where the setup is fully specified by an environment YAML file.  That way we can specify conda-forge for 3.8, which should succeed.

**Comment by quintusdias** [comment]
```
commit bf4e2a6743d7f60ab0e0eb156476fc5c67ad70e1 (HEAD -> devel, origin/devel, issue497)
Author: John Evans <jevans97@utk.edu>
Date:   Mon Dec 9 10:31:30 2019 -0500

    Rewrite Travis-CI interface
    
    There was an issue with conda-forge and poppler that could not be
    resolved and was resulting in failures on all but python 3.8.  Since the
    python versions < 3.8 did not actually need conda-forge, we can go back
    to using the defaults channel, but 3.8 still needs conda-forge.
    
    The 3.6 builds need importlib_resources, but this is not available
    through defaults channel, so we need to use pip for this.  This means
    removing the requirements*.txt files and going full-on with YAML
    configuration files.  This necessitates a big rewrite.
    
    As a final insult, gdal just recently moved to 3.0.2 in defaults, which
    breaks the geotiff test again.  We will require gdal<3.0 for now.
    
    Remove one unneeded shell script from ci.
```
